src/app_service/service/transaction/service.nim

```diff
@@ -209,6 +214,28 @@ QtObject:
     )
     self.threadpool.start(arg)
 
+  proc onFetchDecodedTxData*(self: Service, response: string) {.slot.} =
+    let args = TransactionDecodedArgs()
+    try:
+      let data = parseJson(response)
+      if data.hasKey("result"):
+        args.dataDecoded = $data["result"]
```

Comment:
@Cuteivist are you sure that data will be set properly if `args` is defined with `let`?

Reply:
I admit, got confused with let in JS :). Although interesting part is it did run fine, maybe something did rebuild.

I updated it to `var` and did rebuild, works fine. Thanks @saledjenic